README.md

```diff
@@ -249,6 +256,22 @@ postgresql_database 'vacuum databases' do
 end
 ```
 
+```ruby
+# Create, Insert, Query a SQLite database
+# Note that inserting anything in to the database will create it automaticly.
+sqlite_database 'mr_softie' do
+  connection '/path/to/database.db3'
```

Comment:
Name probably does make more sense.  As we try to abstract the various DBs we should aim to make it as common as possible.  They're all very different technologies, so that's not 100% possible, but where it can be done we should.